### PR TITLE
adding neeraj

### PR DESCRIPTION
**Pull request by neeraj-a-22**
Adding a new user Neeraj

### CODE REVIEW

File: argocd/neeraj-a-22-project.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# apiVersion: argoproj.io/v1alpha1
```
Comment (alexmt) [inline argocd/neeraj-a-22-project.yaml:1]: please uncomment content

**Review by alexmt — CHANGES_REQUESTED** [review]

File: argocd/neeraj-a-22-project.yaml (hunk 1 above)
Comment (neeraj-a-22) [reply argocd/neeraj-a-22-project.yaml]: uncommented please review

File: argocd/neeraj-a-22-project.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: AppProject
+metadata:
+  name: <neeraj-a-22>
```
Comment (jessesuen) [inline argocd/neeraj-a-22-project.yaml:4]: Please remove carats

File: argocd/neeraj-a-22-project.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+spec:
+  sourceRepos:
+    - '*'
+  destinations:
+    - namespace: 'service-<neeraj-a-22>-*'
```
Comment (jessesuen) [inline argocd/neeraj-a-22-project.yaml:9]: remov carats

File: argocd/neeraj-a-22-project.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+      server: http://cluster-argocon:8001
+  roles:
+  - name: admin
+    policies:
+    - p, proj:<neeraj-a-22>:admin, *, *, <neeraj-a-22>/*, allow
+    groups:
+    - <neeraj.agrawal@earnest.com> please copy it from https://argocon.cd.akuity.cloud/user-info
```
Comment (jessesuen) [inline argocd/neeraj-a-22-project.yaml:16]: remove carats

**Review by jessesuen — CHANGES_REQUESTED** [review]

File: argocd/neeraj-a-22-project.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    - '*'
+  destinations:
+    - namespace: 'service-neeraj-a-22-*'
+      server: http://cluster-argocon:8001
+  roles:
+  - name: admin
+    policies:
+    - p, proj:neeraj-a-22:admin, *, *, neeraj-a-22/*, allow
+    groups:
+    - neeraj.agrawal@earnest.com please copy it from https://argocon.cd.akuity.cloud/user-info
```
Comment (jessesuen) [inline argocd/neeraj-a-22-project.yaml]: Please remove trailing text ` please copy it from https://argocon.cd.akuity.cloud/user-info`

**Review by jessesuen — APPROVED** [review]

**Pull request merged**